app/src/main/java/org/sagebionetworks/research/mpower/tracking/view_model/SimpleTrackingItemLog.java

```diff
@@ -0,0 +1,28 @@
+package org.sagebionetworks.research.mpower.tracking.view_model;
+
+import android.support.annotation.NonNull;
+
+import com.google.auto.value.AutoValue;
+
+import org.sagebionetworks.research.mpower.tracking.model.TrackingItem;
+import org.threeten.bp.Instant;
+
+@AutoValue
+public abstract class SimpleTrackingItemLog implements TrackingItemLog {
```

Comment:
iOS team has been pretty strict about documentation for the research framework because it is going to be public facing.  I think we should follow suite with at least mandatory class headers.